app/controllers/experiments_controller.rb

```diff
@@ -435,11 +435,12 @@ def sidebar
   end
 
   def experiment_filter
-    project = Project.readable_by_user(current_user).find_by(id: params[:project_id])
+    project = Project.managable_by_user(current_user).find_by(id: params[:project_id])
```

Comment:
user can be just a viewer on a project but be assigned with management permissions to some experiments